python/lsst/summit/utils/tmaUtils.py

```diff
@@ -258,6 +363,55 @@ def tickFormatter(value, tick_number):
     ax2.xaxis.set_major_locator(mdates.AutoDateLocator())
     ax2.xaxis.set_major_formatter(mdates.DateFormatter('%H:%M:%S'))
 
+    if event.type.name == 'TRACKING':
+        # returns a copy
+        clippedAzimuthData = clipDataToEvent(azimuthData, event, postPadding=TRACKING_RESIDUAL_TAIL_CLIP)
+        clippedElevationData = clipDataToEvent(elevationData, event, postPadding=TRACKING_RESIDUAL_TAIL_CLIP)
+
+        azError = clippedAzimuthData['azError'].values
+        elError = clippedElevationData['elError'].values
+        elVals = clippedElevationData['actualPosition'].values
+        if doFilterResiduals:
+            # Filtering out bad values
+            nReplacedAz = filterBadValues(azError, maxDelta)
```

Comment:
Would it be useful to label/highlight/plot the replaced points in a different colour?